gcloud-java-examples/README.md

```diff
@@ -89,28 +89,22 @@ To run examples from your command line:
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="download <bucket_name> test.txt"
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="delete <bucket_name> test.txt"
     ```
+
  * Here's an example run of `Stat`, illustrating the use of the gcloud-java-nio jar.
 
-     Before running the example, go to the [Google Developers Console][developers-console] to ensure that Google Cloud Storage API is enabled and that you have a bucket with a file in it.
-     Compile the JAR with:
-    ```
-    mvn package -DskipTests -Dmaven.javadoc.skip=true  -Dmaven.source.skip=true
+    Before running the example, go to the [Google Developers Console][developers-console] to ensure that Google Cloud Storage API is enabled and that you have a bucket with a file in it.
+    Run the sample with:
     ```
-    Then run the sample with:
-    ```
-    java -cp gcloud-java-contrib/gcloud-java-nio/target/gcloud-java-nio-0.1.6-SNAPSHOT-shaded.jar:gcloud-java-examples/target/gcloud-java-examples-0.1.6-SNAPSHOT.jar com.google.gcloud.examples.nio.Stat --check
+    mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.nio.Stat" -Dexec.args="--check"
```

Comment:
that's the case 1. I did warn you it was *very* similar! 

The idea is that we'll have examples in both gcloud-java-examples and gcloud-java-nio-example. The former have gcloud-java-nio as a dependency, the latter add its jar in the classpath.